README.md

```diff
@@ -1,61 +1,123 @@
-# Furyagent v0.2.3
+<h1 align="center">
+  <img src="docs/assets/furyagent.png" width="200px"/><br/>
+  Furyagent
+</h1>
+
+<p align="center">Automate some of the Kubernetes cluster management burocracy</p>
 
 [![Build Status](http://ci.sighup.io/api/badges/sighupio/furyagent/status.svg?ref=refs/tags/v0.2.3)](http://ci.sighup.io/sighupio/furyagent)
+![Release](https://img.shields.io/github/v/release/sighupio/furyagent?label=Release)
+![License](https://img.shields.io/github/license/sighupio/furyagent)
+[![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)](https://kubernetes.slack.com/archives/C0154HYTAQH)
 
-## Install
+<!-- <KFD-DOCS> -->
 
-You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+**Furyagent** is a simple CLI tool to automate some common tasks in a Kubernetes cluster management.
 
-Supported architectures are (64 bit):
+Furyagent can help you to:
 
--   `linux`
--   `darwin`
+1. Backup and restore etcd
+2. Dispatch etcd and master certificates
+3. Manage OpenVPN access
+4. Manage SSH access
 
-Download right binary for your architecture and add it to your PATH. Assuming it's downloaded in your
-`~/Downloads` folder, you can run following commands (replacing `{arch}` with your architecture):
+## Installation
 
-```shell
-chmod +x  ~/Downloads/furyagent-{arch}-amd64 && mv ~/Downloads/furyagent-{arch}-amd64 /usr/local/bin/furyagent
+You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+
+To download the latest release, run:
+
+```bash
+wget -q https://github.com/sighupio/furyagent/releases/download/v0.6.1/furyagent-$(uname -s)-amd64 -O /tmp/furyagent
+chmod +x /tmp/furyagent
+sudo mv /tmp/furyagent /usr/local/bin/furyagent
 ```
 
-If you are using MacOS you can also install `furyagent` using the `brew` package manager:
+Alternatively, [Homebrew](https://brew.sh/) users can use `brew` to install `furyagent`:
 
-```shell
+```bash
 brew tap sighupio/furyagent
 brew install furyagent
 ```
 
-To be able to use openvpn client commands, you need also openvpn installed in your system.
+Check that everything is working correctly with `furyagent version`:
+
+```bash
+➜ furyagent version            
+Furyagent version 0.2.3 - md5: 923428f7616f2942b08e2ce7a4d49aa0 - /usr/local/bin/furyagent 
+```
+
+### OpenVPN
 
-For example on MacOS:
+Since Furyagent calls `openvpn` client commands, `openvpn` must be installed in the system.
 
-```shell
+[Homebrew](https://brew.sh/) users can use `brew` to install it:
+
+```bash
 brew install openvpn
 ```
 
-And put 
+Refer to [openvpn-website][openvpn-website] for instructions to install it via other methods.
 
-```
-export PATH=$(brew --prefix openvpn)/sbin:$PATH
-```
+> Remeber to add `openvpn` to your `$PATH` by adding to your `.bashrc` or `.zshrc`:
+>
+> `export PATH=$(brew --prefix openvpn)/sbin:$PATH`
 
-in your .zshrc or .bashrc .
+## Storage
```

Comment:
I would move this section inside "Configuration"

Reply:
✅